acceptance/README.md

```diff
@@ -6,91 +6,94 @@ This directory is intended to manage acceptance testing for the Puppet Server.
+test. You can test an existing build or test a new build with local changes.
 
-##### Testing an existing build
-1. Go to http://builds.delivery.puppetlabs.net/puppetserver/
-2. Scroll down to the most recent build at the bottom.  This will look like:
-   `2.2.2.master.SNAPSHOT.2016.02.18T1627`
-3. Copy the text (not the link address) and set `PACKAGE_BUILD_VERSION` to this
+### PACKAGE_BUILD_VERSION when testing an existing build
+1. Go to http://builds.delivery.puppetlabs.net/puppetserver/?C=M;O=D
+2. This puts the most recent build at the top of the screen. It should
+   look something like: `2.2.2.master.SNAPSHOT.2016.02.18T1627`
+3. Copy the text (not the link address) and set `PACKAGE_BUILD_VERSION` to
+   to this value.
 
-##### Testing local changes
-1. run `lein with-profile ezbake ezbake build`. This will take a while, and
-   will result in a build appearing on http://builds.delivery.puppetlabs.net/puppetserver/
-2. If successful, it will give you back a URL with the new puppetserver version
-   at the end, which will look something like `2.2.2.master.SNAPSHOT.2016.02.18T1627`.
-   Set `PACKAGE_BUILD_VERSION` to this version.
+### PACKAGE_BUILD_VERSION when testing a new build with local changes
+1. Run `lein with-profile ezbake ezbake build`. This will take a while, and
+   will result in a build appearing on http://builds.delivery.puppetlabs.net/puppetserver.
+   2. If successful, it will give you back a URL with the new puppet-server
+   version at the end, which will look something like `2.2.2.master.SNAPSHOT.2016.02.18T1627`.
+   Set `PACKAGE_BUILD_VERSION` to this value.
 
-##### PUPPET_VERSION
+### PUPPET_VERSION
 Define `PUPPET_VERSION` as the packaged version of the puppet-agent package
-that your version of puppetserver depends upon. E.g. 1.3.5
+upon which your version of puppet-server depends, such as 1.3.5. You can pick
+a puppet-agent version from http://builds.delivery.puppetlabs.net/puppet-agent/?C=M;O=D.
 
 If you are testing backwards compatibility against a specific release of
-puppet, the `PUPPET_LEGACY_VERSION` environment variable is available for this
+Puppet, the `PUPPET_LEGACY_VERSION` environment variable is available for this
 purpose, but is not required.
 
-#### Install dependencies
+### Install dependencies
+
+Beaker needs to be installed and is expressed as a Gemfile dependency. to install
+beaker, simply run `bundle install --path .bundle/gems` from the repo's root directory.
+Running beaker will then be a matter of saying `bundle exec beaker _mumble blarg baz_`
+as described next or using the handy shell script created for your testing convenience.
+
+## Running beaker directly
+
+### Create a host config file
 
-Beaker needs to be installed and is expressed as a Gemfile dependency.  To
-install beaker, simply run `bundle install --path .bundle/gems` from the top
-level directory of the project.
+Copy the example RHEL7 host config at
```

Comment:
If I understand your question, I copy it because I don't want to modify the repo version and listen to git whine about it. And I don't want to .`gitignore` the repo version, either.